### PR TITLE
Updated example for using log_reference api in whylabs_client

### PR DESCRIPTION
**Pull request by jamie256**
Adding an updated example for using log_reference api in whylabs_client that extends our basic Whylabs platform example.

### CODE REVIEW

File: python/WhyLabs_Platform_with_log_reference.ipynb [hunk 1]
```diff
@@ -0,0 +1,718 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "834d6471-8490-48ea-bb52-6be31662dc97",
+   "metadata": {},
+   "source": [
+    "# Configure whylogs\n",
+    "\n",
+    "`whylogs`, by default, does not send statistics to WhyLabs.\n",
+    "\n",
+    "There are a few small steps you need to set up. If you haven't got the access key, please onboard with WhyLabs.\n",
```
Comment (richard-rogers) [inline python/WhyLabs_Platform_with_log_reference.ipynb]: Perhaps add a link to onboarding

File: python/WhyLabs_Platform_with_log_reference.ipynb [hunk 2]
```diff
@@ -0,0 +1,718 @@
...
+    "reference_profile_alias = \"demo-reference-profile\"\n",
+    "reference_profile.to_summary()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7ad55e9e",
+   "metadata": {},
+   "source": [
+    "The reference profile can be uploading using a whylabs_client directly, first we need to reference the profile as a file on disk, so write it out."
```
Comment (richard-rogers) [inline python/WhyLabs_Platform_with_log_reference.ipynb]: can be upload_ed_
directly. First, we need...

File: python/WhyLabs_Platform_with_log_reference.ipynb (hunk 1 above)
Comment (jamie256) [reply python/WhyLabs_Platform_with_log_reference.ipynb]: done

File: python/WhyLabs_Platform_with_log_reference.ipynb (hunk 2 above)
Comment (jamie256) [reply python/WhyLabs_Platform_with_log_reference.ipynb]: yeah good catch, fixed.

**Pull request merged**